internal/provider/inmemory/inmemory.go

```diff
@@ -91,5 +91,7 @@ func (i InMemoryDNSProvider) ProviderSpecific() provider.ProviderSpecificLabels
 // Register this Provider with the provider factory
 func init() {
 	client = inmemory.NewInMemoryClient()
-	provider.RegisterProvider("inmemory", NewProviderFromSecret)
+	//ToDo mnairn: This is temporarily being registered as a default to allow time for compatibility changes to be made
+	// to the kuadrant operator dev/test deployment.
+	provider.RegisterProvider("inmemory", NewProviderFromSecret, true)
```

Comment:
so once this merges, we can set the startup value in kuadrant to include `inmemory`, and then change this to false?

Reply:
Exactly. If i set this to false now, which is what we really want it to be, it will potentially break the inmemory provider in the[ kuadrant operator tests ](https://github.com/Kuadrant/kuadrant-operator/pull/658) until it's updated. That said, that's not merged yet :thinking: , i could just make those changes as part of that PR after this is merged.

Reply:
Change of plan, will merge this first as we want it and update the kuadrant operator PR accordingly. Set this to false now, default is only "aws" and "google" now.